src/olympia/api/paginator.py

```diff
@@ -49,3 +52,36 @@ def page(self, number):
                 raise EmptyPage('That page contains no results')
 
         return page
+
+
+class CustomPageNumberPagination(PageNumberPagination):
```

Comment:
This needs a different name to make it explicit that it only works with `ES` (since `ESPaginator` only works with ES "querysets": it calls `.execute()`, accesses the `.total` property, etc.)